creator-node/src/services/prometheusMonitoring/prometheus.constants.ts

```diff
@@ -388,6 +395,23 @@ export const METRICS: Record<string, Metric> = Object.freeze({
       }
     ])
   ),
+  // Add gauge for each monitor
+  ...Object.fromEntries(
+    Object.keys(PROMETHEUS_MONITORS).map((monitor) => {
+      return [
+        METRIC_NAMES[`MONITOR_${monitor}`],
+        {
+          metricType: METRIC_TYPES.GAUGE,
+          metricConfig: {
+            name: METRIC_NAMES[`MONITOR_${monitor}`],
+            help: `Record monitor: ${monitor}`,
+            labelNames: [],
+            aggregator: 'sum' as AggregatorType
```

Comment:
I think `sum` should work since it'll be 0 on all other workers and only have a value on 1 worker. if there's any issue we can try `max` instead

Reply:
i tried sum and max and they both seem to work and only run on worker 1. i left as max since it follows your convention of "If it's a metric returned by state machine, use 'max' because only one worker listens for state machine onComplete and subsequently updates each job's metrics. "